lib/pure/parsesql.nim

```diff
@@ -1424,7 +1424,7 @@ proc ra(n: SqlNode, s: var SqlWriter) =
     for i in 1..n.len-1:
       if i > 1: s.add(',')
       ra(n.sons[i], s)
-    s.add(");")
```

Comment:
This changes the existing behaviour.

Reply:
the extro semicolon for what , I dont like double semicolon at statement end.